### ISSUE TITLE
Dates are not necessarily in consecutive order when loading files

### ISSUE THREAD

**Issue by jetesdal** [ECCO-GROUP/ECCOv4-py#94]
Following code gives me an xarray dataset with unsorted time points (see output below)
```
data_dir= ECCO_dir + '/nctiles_monthly_snapshots'

year_start = 1993
year_end = 2017

# load one extra year worth of snapshots
ecco.recursive_load_ecco_var_from_years_nc(data_dir,vars_to_load=['ETAN'],years_to_load=range(year_start, year_end+1))
```

```
print(ecco_monthly_snaps.ETAN.time.isel(time=[0, -1]).values)
```
['2008-01-01T00:00:00.000000000' '1996-01-01T00:00:00.000000000']

I think the problem is here:
https://github.com/ECCO-GROUP/ECCOv4-py/blob/ed753d27846b21fcc4ef0ddee937f3618be9fc6e/ecco_v4_py/tile_io.py#L211

and it probably can be fixed by adding sorted():
```
files = sorted(list(var_path.glob('**/*nc')))
```

**Comment by timothyas** [comment]
Thanks @jetesdal this is good to know. I have to admit, I have not used (or tested!) this function very much myself. And yes, while the array can be sorted after the fact, I agree it would be nice to load things in order from the get go ( and this is what we're all expecting - time should go forward.....).

Have you tested this out with your own ECCO download?

**Comment by jetesdal** [comment]
Hi @timothyas! Yes, I downloaded the file as described here:
https://ecco-v4-python-tutorial.readthedocs.io/Downloading_the_ECCO_v4_state_estimate.html#ecco-v4-r3-on-po-daac-drive-and-ut-austin-ftp-mirror

And I am trying to run the code in this notebook:
https://ecco-v4-python-tutorial.readthedocs.io/ECCO_v4_Volume_budget_closure.html#Load-2D-MONTHLY-\eta-snapshots

I'm not sure why `var_path.glob('**/*nc')` gives me an unsorted list of files. Maybe this is platform specific. I was able to ensure that the dates are in the right order by sorting the list using `sorted()`.

**Comment by timothyas** [comment]
Closed by #96
